.travis.yml

```diff
@@ -4,6 +4,14 @@ services:
   - docker
 sudo: false
 
+env:
+  global:
+    - REMOTE_NAME: upstream
```

Comment:
I like the configurability here in general. It improves the readability of what follows, as well as of course making it more flexible. As a small point, is it necessary to make `REMOTE_NAME` configurable? Setting as an internal constant would shorten the list of “knobs” that one has to understand to use this. Just a thought. If there is a use case for a custom `REMOTE_NAME` that I’m not seeing them of course it should remain.

Reply:
It was used in two places, so I decided to name it once there. I don't mind to just hard-code it as we should never collide with the existing names on travis.

Reply:
Addressed via https://github.com/NSLS-II/lightsource2-recipes/pull/793/commits/9b9cb32d47d8928b776a75c2e6bfb5cdc6ad69c3.